src/fsharp/FSharp.Core.Unittests/SurfaceArea.coreclr.fs

```diff
@@ -293,6 +293,222 @@ Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityC
 Microsoft.FSharp.Collections.HashIdentity: System.Collections.Generic.IEqualityComparer`1[T] Structural[T]()
 Microsoft.FSharp.Collections.HashIdentity: System.String ToString()
 Microsoft.FSharp.Collections.HashIdentity: System.Type GetType()
+Microsoft.FSharp.Collections.ISeqModule+Core+Activity: Boolean Equals(System.Object)
```

Comment:
Well we could get away with none of it past the ISeq defintion in the interface file. We haven't for two reasons. The first was that when this was does as a single PR it modified too many things, so it was split from a practical engineering reason, rather than a end user reason. The second is that the inlining of these functions does make a difference, and it is what otherwise gives Nessos' Streams the edge. This can be seen in [@mrange's post](https://gist.github.com/mrange/09ba6d7541468de49426eeb81a8b660c) - the difference between SeqComposer and SeqComposer2, as the names stood at that point in time.